src/controller/java/src/chip/devicecontroller/ICDCheckInDelegateWrapper.java

```diff
@@ -67,7 +67,7 @@ private byte[] onKeyRefreshNeeded(
   }
 
   @SuppressWarnings("unused")
-  private void onKeyRefreshDone(int errorCode) {
+  private void onKeyRefreshDone(Long errorCode) {
```

Comment:
@yunhanw-google 
If signature is "(J)V", it should be "long", not "Long".

Reply:
good catch, thanks